centaur/src/main/resources/standardTestCases/retry_with_more_memory/retry_same_memory_output_failure.wdl

```diff
@@ -0,0 +1,27 @@
+version 1.0
+
+task imitate_oom_error {
+  command {
+    # This task mimics printing an exception that would cause it to retry with increased memory, but
+    # because the return code is 1 and continueOnReturnCode is true, the task does not retry with more memory.
+    printf "Exception in thread "main" java.lang.OutOfMemoryError: testing\n\tat Test.main(Test.java:1)\n" >&2 && (exit 1)
+    # As a simulation of an OOM condition, do not create the 'foo' file. Cromwell should still be able to delocalize important detritus.
+    # touch foo
+  }
+  output {
+    # Since this file does not exist, the code will go through an exception failure path. The task will end up
+    # retrying, but memory does not increase.
+    File foo = "foo"
```

Comment:
I was never able to find the code where the status is set to "Failed" because of this incorrect output. If you know where it is, please let me know!

Reply:
I'll make a note-to-self to try to track it down

Reply:
I think the chain is roughly:
* This file doesn't exist
* When we try to delocalize it, that therefore fails
* Therefore, the PAPI result is a failure
* When we interpret the PAPI metadata for the run, we use this method ([link](https://github.com/broadinstitute/cromwell/blob/develop/supportedBackends/google/pipelines/v2alpha1/src/main/scala/cromwell/backend/google/pipelines/v2alpha1/api/request/GetRequestHandler.scala#L48)) which interprets this as a Failed status, which the actor system then records as a failure

Reply:
Thanks @cjllanwarne!